ci/concourse/terraform-dockerimage-pipeline.yml

```diff
@@ -0,0 +1,191 @@
+resources:
+    - name: terraform-ibm-ha-repo
+      type: git
+      icon: github
+      source:
+          uri: https://github.com/fshuva/ibm-fortigate-AP-HA-terraform-deploy.git
+          branch: concourse_setup
+    - name: terraform-0-13-7
```

Comment:
don't think you need to construct the name with the version unless you use more than 1 version in a single pipeline.
can simply name it terraform-tool or terraform-image